beater/test_approved_es_documents/TestPublishIntegrationErrors.approved.json

```diff
@@ -295,6 +311,16 @@
                     "platform": "darwin"
                 }
             },
+            "kubernetes": {
+                "namespace": "namespace1",
+                "node": {
```

Comment:
out of curiosity, why don't we have same fields as ECS (eg we have node, but not container, labels, etc)?

Reply:
I think it's because this is the only information we're able to retrieve from within a container.